src/Cli/dotnet/ReleasePropertyProjectLocator.cs

```diff
@@ -186,15 +185,16 @@ private bool IsValidProjectFilePath(string path)
         }
 
         /// <returns>A case-insensitive dictionary of any properties passed from the user and their values.</returns>
-        private Dictionary<string, string> GetGlobalPropertiesFromUserArgs(IEnumerable<string> userPropertyArgs)
+        private Dictionary<string, string> GetGlobalPropertiesFromUserArgs(ParseResult parseResult)
         {
             Dictionary<string, string> globalProperties = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
-            IEnumerable<(string key, string value)> globalPropEnumerable = MSBuildPropertyParser.ParseProperties(String.Join(";", userPropertyArgs));
 
-            // The parser puts keys into the format --property:Key no matter the user input, so we can expect that pattern and extract the key.
-            foreach (var kvPair in globalPropEnumerable)
+            string[] globalPropEnumerable = parseResult.GetValueForOption(CommonOptions.PropertiesOption);
+
+            foreach (var keyEqVal in globalPropEnumerable)
             {
-                globalProperties[kvPair.key.Split(new string[] { "--property:" }, StringSplitOptions.None).Last()] = kvPair.value;
+                string[] keyValuePair = keyEqVal.Split("=");
```

Comment:
This split concerns me. a property value like `RepositoryUrl="https://repo.mycorp.com/repos?id=321432asdfs=="` would split incorrectly and we'd mangle the value.  I think you should use `keyEqVal.Split("=", 2)` to ensure that only two substrings are returns (i.e. that only the first `=` is split on).

Reply:
Done!